apis/certificatemanager/certificatemanager.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package certificatemanager contains certificatemanager API versions
+package certificatemanager
```

Comment:
It appears that the resource being added here in a *private* CA (i.e. `acmpa` vs `acm`. I think it should likely be isolated in its own group (i.e. `privatecertificatemanager`).

Reply:
Yes, we will make sure to do the changes when merging the both the branches and will raise a isolated PR for this.